proxy/proxy.go

```diff
@@ -148,8 +147,9 @@ func (proxy *Proxy) ListenAndServe() error {
 func (proxy *Proxy) weaveCIDRsFromConfig(config *docker.Config, hostConfig *docker.HostConfig) ([]string, error) {
 	if hostConfig != nil &&
 		hostConfig.NetworkMode != "" &&
+		hostConfig.NetworkMode != "default" &&
```

Comment:
I cannot find a mention of this new mode anywhere in the Docker API or CLI docs. Both list the modes as "bridge, host, and container:<name|id>"

Yet simply starting a container on Docker 1.8 does produce this mode in `docker inspect`. @mavenugo is the code wrong (i.e. "default" shouldn't be there) or the docs?